### ISSUE TITLE
MLG: Non-Cartesian Similarities

### ISSUE THREAD

**Issue by lefnire** [lefnire/ocdevel#22]
[keras.io/api/metrics](https://keras.io/api/metrics/)
* Accuracy metrics
* Regression metrics
* Classification metrics based on True/False positives & negatives
* Image segmentation metrics
* Hinge metrics for "maximum-margin" classification

Other
* Hamming: difference between binary strings (XOR)
* jensen_shannon, jaccard, KL divergence
* [knobs-dials.com](https://helpful.knobs-dials.com/index.php/Similarity_or_distance_measures/metrics)
